docker/jupyterlab/sparkR_k8s/Rstartup

```diff
@@ -8,7 +8,7 @@
   options(defaultPackages = c(old, "SparkR"))
 
   r <- getOption("repos")
-  r["CRAN"] <- "https://cran.uib.no/"
+  r["CRAN"] <- "https://packagemanager.rstudio.com/all/latest"
```

Comment:
I think it's best to change the name from `CRAN` since we're changing the source. Would it be helpful to keep CRAN as a backup too? I believe it's possible to have a list of repos.

Reply:
Ok, I'll use a different name. The default CRAN setting will be blocked by Istio anyway, but I'v tested something else now that seem to work...